src/encoded/__init__.py

```diff
@@ -35,7 +35,7 @@
 # from webob.cookies import JSONSerializer
 
 from .loadxl import load_all
-from .util import find_other_in_pair
+# from .util import find_other_in_pair
```

Comment:
I'm removing this commented-out code and a number of other commented-out exports.